### ISSUE TITLE
[MWCC]: mwcceppc file path behavior broken

### ISSUE THREAD

**Issue by Cuyler36** [decompals/wibo#78]
#75 (wibo 0.6.14+) causes issues with mwcceppc paths that worked in previous versions. This causes issues on all Linux environments. 

Our setup uses assetrip + assetinc to rip binary assets from the source binaries into build/include/assets/... and we have `-i build/include` passed into mwcceppc.exe. Since the issue started occurring with wibo 0.6.14, the issue must be these lines: https://github.com/decompals/wibo/pull/75/files#diff-35916b9fc9415b9ed5c6c9e5cb55c16f9c61754053de1ac138aa4456378610dfR685-R691

Example error:
```
[422/1595] CC build/iconv/src/static/version.c
FAILED: build/src/static/version.o 
/opt/devkitpro/devkitPPC/bin/powerpc-eabi-cpp -M build/iconv/src/static/version.c -MF build/src/static/version.o.d -nostdinc -I include -I tools/ppcdis/include -I build/include -I /N64_SDK/ultra/usr/include && wibo tools/1.3.2/mwcceppc.exe -O4 -char unsigned -fp hard -inline on -enum int -sdata 0 -sdata2 0 -d _LANGUAGE_C -d F3DEX_GBI_2 -d NDEBUG -d DEBUG=0 -d MUST_MATCH -nostdinc -proc gekko -maxerrors 1 -Cpp_exceptions off -I- -i include -i tools/ppcdis/include -i build/include -ir /N64_SDK/ultra/usr/include -c build/iconv/src/static/version.c -o build/src/static/version.o
### mwcceppc.exe Compiler:

User break, cancelled...
#    File: build\iconv\src\static\version.c
# -----------------------------------------
#       4: #include "assets/__Creator__.inc"
#   Error:                                 ^
#   the file 'assets/__Creator__.inc' cannot be opened
#   Too many errors printed, aborting program
[434/1595] Order floats config/rel.yml 806432c0 806432c8
ninja: build stopped: subcommand failed.
Dumping dump/forest_1st.arc
Dumping dump/forest_2nd.arc
Error running Ninja build
```

**Comment by jpburnett** [comment]
Thanks for opening the issue, I'll take a look into this and see what I can find out.

**Comment by MisterSheeple** [comment]
Any updates on this?

**Comment by mkst** [comment]
Have you tried the latest commit?

**Comment by Cuyler36** [comment]
> Have you tried the latest commit?

Unfortunately it did not fix the issue for us. In fact, it appears even more broken than before. Now it can't even find top level include directives. These are in our /include/ directory.

```
[8052/9273] CC src/static/initial_menu.c
FAILED: build/src/static/initial_menu.o 
/opt/devkitpro/devkitPPC/bin/powerpc-eabi-cpp -M src/static/initial_menu.c -MF build/src/static/initial_menu.o.d -nostdinc -I include -I tools/ppcdis/include -I build/include -I ~/n64sdk/ultra/usr/include && wibo tools/sjiswrap.exe tools/1.3.2/mwcceppc.exe -O4 -char unsigned -fp hard -inline on -sdata 0 -sdata2 0 -d _LANGUAGE_C -d F3DEX_GBI_2 -d NDEBUG -d DEBUG=0 -d MUST_MATCH -nostdinc -proc gekko -maxerrors 1 -Cpp_exceptions off -I- -i include -i tools/ppcdis/include -i build/include -ir ~/n64sdk/ultra/usr/include -c src/static/initial_menu.c -o build/src/static/initial_menu.o
### mwcceppc.exe Compiler:

User break, cancelled...
#    File: src\static\initial_menu.c
# ----------------------------------
#       1: #include "initial_menu.h"
#   Error:                         ^
#   the file 'initial_menu.h' cannot be opened
#   Too many errors printed, aborting program
```

**Comment by Cuyler36** [comment]
I can confirm that reverting to wibo 0.6.13 fixes both the issues with 0.6.14 and 0.6.15.
